.platformio with espidf framework compability

Hello, i noticed that [commit](https://github.com/lvgl/lv_port_esp32/commit/a42c982c912e2f93868ebec21306e7a8c912d2f1) says this project have compability with .platformio build system, but i dont have any luck to compile that. 

For now i have bunch of "undefined reference to `lv_something'" and i don't know how to fix any of those things.

```
Linking .pio\build\lolin32\firmware.elf
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o:(.literal._Z6lvtestPv+0x4): undefined reference to 'lv_disp_get_default'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o:(.literal._Z6lvtestPv+0x8): undefined reference to 'lv_disp_get_scr_act'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o:(.literal._Z6lvtestPv+0xc): undefined reference to 'lv_btn_create'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o:(.literal._Z6lvtestPv+0x10): undefined reference to 'lv_obj_set_pos'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o:(.literal._Z6lvtestPv+0x14): undefined reference to 'lv_obj_set_size'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o:(.literal._Z6lvtestPv+0x18): undefined reference to 'lv_label_create'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o:(.literal._Z6lvtestPv+0x1c): undefined reference to 'lv_label_set_text'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o: in function 'lv_scr_act':
E:\work\projects\esp32-platformio/src\../components/lv_port_esp32/components/lvgl/lvgl/src/lv_core/lv_disp.h:99: undefined reference to 'lv_disp_get_default'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: E:\work\projects\esp32-platformio/src\../components/lv_port_esp32/components/lvgl/lvgl/src/lv_core/lv_disp.h:99: undefined reference to 'lv_disp_get_scr_act'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio\build\lolin32\esp-idf\src\main.cpp.o: in function 'lvtest(void*)':
E:\work\projects\esp32-platformio/src/main.cpp:13: undefined reference to 'lv_btn_create'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: E:\work\projects\esp32-platformio/src/main.cpp:14: undefined reference to 'lv_obj_set_pos'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: E:\work\projects\esp32-platformio/src/main.cpp:15: undefined reference to 'lv_obj_set_size'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: E:\work\projects\esp32-platformio/src/main.cpp:19: undefined reference to 'lv_label_create'
c:/users/dr_zlo/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.2.0/../../../../xtensa-esp32-elf/bin/ld.exe: E:\work\projects\esp32-platformio/src/main.cpp:20: undefined reference to 'lv_label_set_text'
```

--- a/README.md
+++ b/README.md
@@ -268,11 +268,17 @@ Modify the `main.c` like this:
 int app_main(void)
 {
     lv_init();
-    // hw_init();
+
+    /* Initialize your hardware. */
+    
+    /* hw_init(); */
 
     demo_create();
 
-    // hw_loop();
+    /* Create the UI or start a task for it.
+     * In the end, don't forget to call `lv_task_handler` in a loop. */
+
+    /* hw_loop(); */
 
     return 0;
 ```

--- a/README.md
+++ b/README.md
@@ -243,3 +243,38 @@ There are several development boards based on the ESP32 chip, make sure you assi
 
 For development kits that come with a display already attached, all the correct settings are already known and can be selected in `menuconfig` from the first option "Select predefined display configuration." Once selected all the other options will be defaulted and won't appear in the menu.
 
+## Platformio support
+
+Using the [lv_platformio](https://github.com/lvgl/lv_platformio) project add the following lines to `platformio.ini` file:
+
+```
+[env:esp32]
+platform = espressif32
+framework = espidf
+board = esp-wrover-kit
+```
+
+Change the default environment to `default_envs = esp32`.
+
+Modify the `main.c` like this:
+
+```c
+#include "lvgl.h"
+
+// #include "driver.h"
+
+#include "demo.h"
+
+int app_main(void)
+{
+    lv_init();
+    // hw_init();
+
+    demo_create();
+
+    // hw_loop();
+
+    return 0;
+```
+
+For more information see: [platformio with espidf framework compability](https://github.com/lvgl/lv_port_esp32/issues/168).

